docker/spec/fixtures/github/contents_helm_repo.json

```diff
@@ -0,0 +1,227 @@
+      "type": "file",
+      "_links": {
+        "self": "https://api.github.com/repos/dependabot/dependabot-core/contents/dependabot-core.gemspec?ref=master",
+        "git": "https://api.github.com/repos/dependabot/dependabot-core/git/blobs/70ca6dcb9506728ca235748d27fb871497c349a0",
+        "html": "https://github.com/dependabot/dependabot-core/blob/main/dependabot-core.gemspec"
+      }
+    },
+    {
+      "name": "helpers",
+      "path": "helpers",
+      "sha": "900f6c83a9611c92478918dfca8e541261912e8c",
+      "size": 0,
+      "url": "https://api.github.com/repos/dependabot/dependabot-core/contents/helpers?ref=master",
+      "html_url": "https://github.com/dependabot/dependabot-core/tree/master/helpers",
+      "git_url": "https://api.github.com/repos/dependabot/dependabot-core/git/trees/900f6c83a9611c92478918dfca8e541261912e8c",
+      "download_url": null,
+      "type": "dir",
+      "_links": {
+        "self": "https://api.github.com/repos/dependabot/dependabot-core/contents/helpers?ref=master",
+        "git": "https://api.github.com/repos/dependabot/dependabot-core/git/trees/900f6c83a9611c92478918dfca8e541261912e8c",
+        "html": "https://github.com/dependabot/dependabot-core/tree/master/helpers"
+      }
+    },
+    {
+      "name": "lib",
+      "path": "lib",
+      "sha": "8493ef79eb42da9d8aaa1e78dab729e3c08d22ee",
+      "size": 0,
+      "url": "https://api.github.com/repos/dependabot/dependabot-core/contents/lib?ref=master",
+      "html_url": "https://github.com/dependabot/dependabot-core/tree/master/lib",
+      "git_url": "https://api.github.com/repos/dependabot/dependabot-core/git/trees/8493ef79eb42da9d8aaa1e78dab729e3c08d22ee",
+      "download_url": null,
+      "type": "dir",
+      "_links": {
+        "self": "https://api.github.com/repos/dependabot/dependabot-core/contents/lib?ref=master",
+        "git": "https://api.github.com/repos/dependabot/dependabot-core/git/trees/8493ef79eb42da9d8aaa1e78dab729e3c08d22ee",
+        "html": "https://github.com/dependabot/dependabot-core/tree/master/lib"
+      }
+    },
+    {
+      "name": "spec",
+      "path": "spec",
+      "sha": "40e3fc7a0819fac4c3e30b0d4217ad11ffc2ceef",
+      "size": 0,
+      "url": "https://api.github.com/repos/dependabot/dependabot-core/contents/spec?ref=master",
+      "html_url": "https://github.com/dependabot/dependabot-core/tree/master/spec",
+      "git_url": "https://api.github.com/repos/dependabot/dependabot-core/git/trees/40e3fc7a0819fac4c3e30b0d4217ad11ffc2ceef",
+      "download_url": null,
+      "type": "dir",
+      "_links": {
+        "self": "https://api.github.com/repos/dependabot/dependabot-core/contents/spec?ref=master",
+        "git": "https://api.github.com/repos/dependabot/dependabot-core/git/trees/40e3fc7a0819fac4c3e30b0d4217ad11ffc2ceef",
+        "html": "https://github.com/dependabot/dependabot-core/tree/master/spec"
+      }
+    }
+  ]
+  
```

Comment:
Does helm generate these files without a final newline for some reason? I see this is true of several other files as well. 

If _not_ auto generated by Helm this way, can you please fix?

Reply:
Fixed.

Reply:
Thanks. I filed https://github.com/dependabot/dependabot-core/issues/5814 to add a linter for this to speed up the feedback loop.

Reply:
Actually, you mention fixed, but I still see missing newlines on this file and the other json file? Did you forget to `git add` them perhaps?

Reply:
So the newlines were there, but I think that because the JSON wasn't fully formatted the newline detector was confused.

I think I have fixed this now.